241 Implement new Figma designs from Sept/Oct 2025

chrislng opened the pull request:
Implemented new badges, new collection design, mascots.

<img width="278" height="558" alt="Screenshot 2025-11-28 at 3 36 30 AM" src="https://github.com/user-attachments/assets/90a7ab61-3ab5-438e-a5b5-7fd5d959237a" />
<img width="279" height="566" alt="Screenshot 2025-11-28 at 3 36 45 AM" src="https://github.com/user-attachments/assets/d65e2abd-e105-48a0-ad37-39b9a6d38d11" />
<img width="276" height="561" alt="Screenshot 2025-11-28 at 3 38 11 AM" src="https://github.com/user-attachments/assets/25967a26-8e64-4f69-9944-a7a41405f973" />
<img width="282" height="565" alt="Screenshot 2025-11-28 at 3 38 23 AM" src="https://github.com/user-attachments/assets/7953efe6-d15f-487b-8f27-d4183911f0ad" />
<img width="283" height="565" alt="Screenshot 2025-11-28 at 3 39 10 AM" src="https://github.com/user-attachments/assets/cc4563db-fd16-412b-91fe-ebc953a6f683" />
<img width="286" height="572" alt="Screenshot 2025-11-28 at 3 39 05 AM" src="https://github.com/user-attachments/assets/f06f1b14-e59f-4f16-a357-744644673280" />
<img width="279" height="563" alt="Screenshot 2025-11-28 at 3 38 56 AM" src="https://github.com/user-attachments/assets/7e8303a6-4462-4e6e-9e7c-bc748dccc1a8" />
<img width="266" height="551" alt="Screenshot 2025-11-28 at 5 33 30 AM" src="https://github.com/user-attachments/assets/3aa54add-b3ba-4fb4-aa63-cc8488b0bd71" />


Closes #241 , closes #283, closes #282

chrislng committed:
New badge page layout from figma

New black badges (some placeholders)

Fixed plural/sigular in CollectionPage.vue

chrislng committed:
Implemented round badge image color logic

Depending on if not in progress, in progress or unlocked.

Modals still use the old locked/unlocked colors.

chrislng committed:
Complete round badges implementation

And gitignore update to ignore Vite timestamp files

chrislng committed:
Refactor location handling: integrate LocationService

chrislng committed:
Implement changes from Figma

chrislng committed:
Remove /black/ and /unlocked-grid/ files since unneeded.

chrislng committed:
Collection and Badges: smaller gap, change Trophy title, implement Figma tab style

Remove counters, add collection and trophy count in parentheses next to tab names

chrislng committed:
Add new trophy badge images

chrislng committed:
Implement collection/badge toggle and new avatar icon from Figma

chrislng committed:
Implement square, titleless collection grid from Figma

chrislng committed:
Set inactive collection/badge toggle section title color to grey

JanssenSimon commented:
Il faudrait que tu règle le mini merge conflict trivial avant de pouvoir merge, @chrislng

chrislng committed:
Implement new badge figma design, new territory Rimouski badge

chrislng committed:
Add unit tests

chrislng committed:
Fix typo

chrislng committed:
Handle new figma progress badges

chrislng committed:
Remove unused TopToolbar.css import and correct typo

chrislng committed:
Implement new category, trophy and brough sections from Figma

chrislng committed:
Badges - Change css to better match figma

chrislng committed:
Font changes and always use "progress" badge version in discovery modal

chrislng committed:
Add mascot images

chrislng committed:
Add mascot image to discovery details modal

chrislng committed:
Implementé page collection vide avec mascotte depuis Figma

chrislng commented:
Closes #281

chrislng committed:
Implement badge modal change from Figma

-At the bottom instead of at the center
-Progress bar
-Refactored the badge modal in its own file to avoid code duplication

JanssenSimon reviewed (approved):
J'adore les nouveaux visuels
